src/view/event-view.js

```diff
@@ -1,33 +1,63 @@
 import {createElement} from '../render.js';
+import {humanizeEventDate, calculateDateDif} from '../utils.js';
 
-const createEventTemplate = () => (
-  `<li class="trip-events__item">
+const createOfferTemplate = ({title, price}) => (
+  `<li class="event__offer">
+    <span class="event__offer-title">${title}</span>
+    &plus;&euro;&nbsp;
+    <span class="event__offer-price">${price}</span>
+  </li>`
+);
+
+const getOffersTemplate = (offers) => (
+  offers
+    .map((offer) => createOfferTemplate(offer))
+    .join('')
+);
+
+const createEventTemplate = (event) => {
+  const {
+    basePrice,
+    dateFrom,
+    dateTo,
+    destination,
+    isFavorite,
+    offers,
+    type
+  } = event;
+
+  const dateFromToDate = humanizeEventDate(dateFrom,'YYYY-MM-DD');
+  const dateFromToTitle = humanizeEventDate(dateFrom,'DD MMM');
+  const dateFromToDateTime = humanizeEventDate(dateFrom,'YYYY-MM-DD[T]HH:mm');
+  const dateFromToTime = humanizeEventDate(dateFrom,'HH:mm');
+  const dateToToDateTime = humanizeEventDate(dateTo,'YYYY-MM-DD[T]HH:mm');
+  const dateToToTime = humanizeEventDate(dateTo,'HH:mm');
+  const getFavoriteClass = isFavorite ? 'event__favorite-btn--active' : '';
```

Comment:
Здесь хранится результат тернарного оператора. Глаголы следует использовать только в функциях
```suggestion
  const favoriteClass = isFavorite ? 'event__favorite-btn--active' : '';
```

Reply:
принято